_posts/2022-11-11-jberet-intro.md

```diff
@@ -0,0 +1,11 @@
+---
+layout:     post
+title:      "Introducing The Java Batch Processing API And JBeret Implementation"
+subtitle:   ""
+date:       2022-11-01
```

Comment:
is this an old post back in 2022?  Why is it empty?

Reply:
oops, I'll fix it :D